### DISCUSSION TITLE
headers for virtualenv python

### DISCUSSION THREAD

**Discussion by robtaylor** [python-poetry/poetry#7052] [General]
Hi, I'm packaging some existing c++ python modules that make use of boost::python. This requires pyconfig.h. 
Is there a way to make the virtualenv python be installed with dev headers, and if there is how do I find the path?

Thanks!

**Comment by Secrus** [comment]
Poetry is using [`virtualenv`](https://github.com/pypa/virtualenv) for creating its venvs, so I guess you should raise your question with them.
